### ISSUE TITLE
Document the Local Server config options for containers

### ISSUE THREAD

**Issue by roborourke** [humanmade/altis-local-server#257]
After #159 went in we can now configure to some extent the containers that are set up by local server. This hasn't been documented yet so we should cover all of the options in a new page of documentation.

The key element to note here is that the other modules in Altis are not "aware" of any changes made to the configuration of this module yet so the documentation should cover the additional modifications required to prevent errors from coming up. These include:

- XRay container
   - cloud module xray setting should be off
- Elasticsearch container
   - search module should be off
   - ~analytics module "native" setting should be off~
- Kibana (no module dependencies)
- ~Analytics containers~
   - ~analytics module native setting should be off~
- Cavalcade
   - cloud module cavalcade setting should be off

**Comment by ivankristianto** [comment]
@roborourke do you have a sample of how to use this on Travis?
For example during travis tests, we don't need:
- mailhog
- s3
- xray
- pinpoint
- cognito
- tachyon
- cavalcade
- kibana

But this setting only for CI. I wonder if we can exclude this from the composer.json?

**Comment by roborourke** [comment]
I don’t think we allow for turning all of those off yet. There’s some potential crossover with needing to turn the related software off in the cloud module too.

the best way I think is to set the environment type constant returned by `Altis\get_environment_type()` to “ci” when you know it’s on Travis, then you use the config environment overrides with ci as the environment name.

i still need to make that an officially supported thing but I need to check everywhere we use logic that checks the environment type so nothing breaks.

**Comment by ivankristianto** [comment]
Thanks @roborourke that make sense, let me try it on RB
